README.md

```diff
@@ -63,26 +63,6 @@ On Linux and MacOS it works out of the box following the steps.<br/>
   cp sample.env .env
   ```
 
-### .env
-
-The `.env` file contains environment variables that the program needs to function properly. Some of these variables are deployment-specific, so they aren't included in the repository.
-
-- `REACT_APP_PORTIS_DAPP_ID`
-
-  Allows you to connect a Portis wallet. Without this the program will hang after choosing Portis and clicking the "Pair" button. Portis Dapp IDs aren't secret, but they are domain-specific; you can get the one we use for testing by making a post asking for it in the [Discord](https://discord.gg/shapeshift).
-
-- `REACT_APP_ETHEREUM_NODE_URL`
-
-  Needed for certain Defi integrations such as Yearn; the app will malfunction when connecting a wallet without it.
-
-  Any Ethereum node should do, but you can get your own node URL for testing by doing the following:
-
-  1. Go to https://infura.io/dashboard
-  2. Set up a free account
-  3. Make a new project
-
-      Your key should use "JSON-RPC over HTTPS" and look like this: `https://mainnet.infura.io/v3/<your project id>`
-
```

Comment:
maybe we just drop the line `Some of these variables are deployment-specific, so they aren't included in the repository` and add a reference to env.sample? This is still good info, and if someone's deploying something somewhere that isn't app.shapeshift.com or localhost, they'll need their own keys

Reply:
will make this change